src/software/new_geom/util/contains.h

```diff
@@ -1,14 +1,24 @@
 #pragma once
 
 #include "software/new_geom/circle.h"
+#include "software/new_geom/polygon.h"
+#include "software/new_geom/ray.h"
+#include "software/new_geom/rectangle.h"
 #include "software/new_geom/segment.h"
 
 /**
- * Returns whether the Circle contains the Segment
- * Segment is considered contained in Circle if both endpoints are inside the Circle
+ * Returns whether `container` contains `contained`
  *
- * @param out
- * @param in
- * @return true if the Circle contains the Segment, false otherwise
+ * @param container
+ * @param contained
+ *
+ * @return whether `container` contains `contained`
  */
-bool containsNew(const Circle &out, const Segment &in);
+bool contains(const Circle& container, const Segment& contains);
+bool contains(const Circle& container, const Point& contained);
```

Comment:
boost calls it [within](https://www.boost.org/doc/libs/1_62_0/libs/geometry/doc/html/geometry/reference/algorithms/within/within_2.html) where it checks whether the first geom is within the second geom. @akhilveeraghanta  @MathewMacDougall thoughts?

Reply:
`contains` feels a bit more natural to me (I wouldn't normally say "within" when talking about geom), but I'm also bias because I'm used to it being called `conatains`.

@jsn @jared-paul @LiCody ?

Reply:
``` contains ``` sounds more natural to me.

Reply:
Id argue `within` would be better here because normally we are used to seeing `circle.contains(point)` 

out of `if(within(circle, point))` vs `if(contains(circle, point))` id say the first one makes slightly more sense, but i don't feel strongly about this, so if everyone prefers contains thats fine w/ me

Reply:
to be clear it would be `if(within(point, circle))`

Reply:
I think I prefer `contains` as well, I think in my head it's little more clear that it means "fully contains" over "within", but I don't feel that strongly either way.